tensorflow/tensorboard/backend/server.py

```diff
@@ -67,21 +68,19 @@ def ParseEventFilesSpec(logdir):
   files = {}
   if logdir is None:
     return files
+  uri_pattern = re.compile("[^:/]+://.*")
```

Comment:
Thanks for the comments, @jhseu.
I rebased against the latest code and fix this.